initial/src/main/java/com/example/services/PaymentStatusService.java

```diff
@@ -0,0 +1,29 @@
+package com.example.services;
+
+import java.util.*;
+
+public class PaymentStatusService {
+    // payment_status service (id, user_id, boolean is_paid, expiration_date(null or set a date if is_paid is true))
+    public enum PaymentStatus {
+        PAID,
+        UNPAID,
+        PENDING;
```

Comment:
Can you please add *Javadoc* for each enum here?

Reply:
done